src/util/index.js

```diff
@@ -37,3 +37,7 @@ export const validateEmail = email => {
   const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
   return re.test(email.toLowerCase())
 }
+
+export function isDateInTheFuture (date) {
```

Comment:
i think we use to have this and then i think @lorenjohnson moved it to hylo-shared?

Reply:
righto, I think it got dragged back in during the merge deconfliction (which was very hairy for this one...)